contributors.txt

```diff
@@ -2,4 +2,6 @@ Anzt Hartwig <hartwig.anzt@kit.edu> Karlsruhe Institute of Technology
 Cojean Terry <terry.cojean@kit.edu> Karlsruhe Institute of Technology
 Drzaic Jelena <jelena.drzaic1@gmail.com>
 Flegar Goran <flegar@uji.es> Universitat Jaume I
-Nayak Pratik <pratik.nayak@kit.edu> Karlsruhe Institute of Technology
```

Comment:
Lastname Firstname <email> [institution]
without comma, and please ordered alphabetically (according to last name)

Reply:
Fixed